GoogleForCreators/web-stories-wp
# Templates: Remove first page animations

## Context


Remove all the first page animations on current story templates.

## Summary


Sets all templates the first page animations to an empty array.

## Relevant Technical Choices
First page animations are not supported by AMP.  Web Story templates should not include first page animations.


## To-do



## User-facing changes


Before:
'First Page Animations' checklist item
![image](https://user-images.githubusercontent.com/14057928/173427209-8e978166-4755-4fa1-ac13-1ac19f484a64.png)

Disabled Play button
![image](https://user-images.githubusercontent.com/14057928/173427305-9150c482-c6e5-4cc8-9263-3b6f297ed3e3.png)

After:
No 'First Page Animations' checklist item on first page (so no first page animations)
![image](https://user-images.githubusercontent.com/14057928/173427413-0b396efd-0226-4387-bb52-6b6e8d8236fa.png)

No Play button
![image](https://user-images.githubusercontent.com/14057928/173427445-ec5f835f-805b-45d3-b5a7-6d9e1a297839.png)


## Testing Instructions






This PR can be tested by following these steps:

1. Go to Dashboard
2. Choose a template (most if not all of them seem to first page animations)
3. Once in the editor with the chosen template, click the Checklist icon at the bottom of the editor
4. Go to the 'Design' section within the Checklist

On Staging:
- Notice 'First Page Animations' checklist item
- Notice the disabled 'Play' animations button on the canvas

On Branch:
- Notice no 'First Page Animations' checklist item
- Notice no 'Play' animations button on the canvas

## Reviews

### Does this PR have a security-related impact?
No


### Does this PR change what data or activity we track or use?
No


### Does this PR have a legal-related impact?
No


## Checklist




---



Fixes #11648
